code/__HELPERS/fade_into_nothing.dm

```diff
@@ -0,0 +1,12 @@
+/// Deletes the atom with a little fading out animation after a specified time
+/atom/proc/fade_into_nothing(var/atom/target = src, life_time = 5 SECONDS, fade_time = 3 SECONDS)
```

Comment:
If this is to be an atom proc, it probably doesn't need the first argument.
You could alternately make it a global proc and retain the first argument.